client/app/telecom/task/telecom-task.controller.js

```diff
@@ -51,7 +53,7 @@ angular.module("managerApp").controller("TelecomTaskCtrl", function ($scope, $tr
     =            INITIALIZATION            =
     ======================================*/
 
-    var init = function () {
+    self.init = function () {
```

Comment:
Init should be that exposed since it's a one time job and caller is at the same level. Using angular ```this.$onInit = ``` would be a bit more elegant, since you would not have to call the function directly